azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -84,11 +84,18 @@ func resourceLogAnalyticsWorkspace() *schema.Resource {
 					string(operationalinsights.WorkspaceSkuNameEnumPremium),
 					string(operationalinsights.WorkspaceSkuNameEnumStandalone),
 					string(operationalinsights.WorkspaceSkuNameEnumStandard),
+					string(operationalinsights.WorkspaceSkuNameEnumCapacityReservation),
 					"Unlimited", // TODO check if this is actually no longer valid, removed in v28.0.0 of the SDK
 				}, true),
 				DiffSuppressFunc: logAnalyticsLinkedServiceSkuChangeCaseDifference,
 			},
 
+			"capacity_reservation_level": {
```

Comment:
the use of "level" seems vague here, going by https://docs.microsoft.com/en-us/azure/azure-monitor/platform/manage-cost-storage it seems to corraspond to "gb/d" so would this be better as
```suggestion
			"reservation_capcity_in_gb_per_day": {
```
or even
```suggestion
			"reservation_capcity": {
```
WDYT?

Reply:
I was torn with `capacity_reservation_level` on first implementation and couldn't come up with a better one at the time. One of the hardest things to do in programming is properly name things 😅 .

That being said I think your suggestion of `reservation_capcity_in_gb_per_day` is a perfect balance. It simplifies it but allows users to not have to pull up the documentation to find out that it is in GB per day 😄 .